src/student/commands/schedule-monitoring.command.ts

```diff
@@ -58,11 +59,15 @@ export class ScheduleMonitoringCommand {
 
     await this.checkStudentSchedules(userId, start, end);
 
-    const topic = await this.prisma.scheduleTopics.findUnique({
-      where: { id: topicId },
-    });
+    let topic: ScheduleTopics | undefined;
+
+    if (typeof topicId !== 'undefined') {
+      topic = await this.prisma.scheduleTopics.findUnique({
+        where: { id: topicId },
+      });
+    }
```

Comment:
Não fica melhor assim?

```suggestion
    let topic: ScheduleTopics;
    if (topicId) {
      topic = await this.prisma.scheduleTopics.findUnique({
        where: { id: topicId },
      });
    }
```